### ISSUE TITLE
Jump-to nav needs margins applied

### ISSUE THREAD

**Issue by gdoyle1** [redhat-developer/rhd-frontend#343]
The jump-to nav is missing 64px margins
<img width="1444" alt="Screen Shot 2019-10-14 at 11 33 17 AM" src="https://user-images.githubusercontent.com/30870144/66763794-7cde3600-ee76-11e9-99c4-3a501c4f228f.png">

**Comment by mindreeper2420** [comment]
@gdoyle1 is this supposed to be closed?

**Comment by gdoyle1** [comment]
@mindreeper2420 Yeah it was fixed - I created another issue that is related to this for mobile views! #326
